### ISSUE TITLE
Hacking-Git: possible additions

### ISSUE THREAD

**Issue by sivaraam** [git/git.github.io#439]
In [Hacking Git](https://git.github.io/Hacking-Git/), the links to the following could possibly be added.

* [maintain-git](https://github.com/git/git/blob/master/Documentation/howto/maintain-git.txt): this has information about the workflow of the project
    * Another related addition: "[Large-Merging Workflows](https://git-scm.com/book/en/v2/Distributed-Git-Maintaining-a-Project#_large_merging_workflows)" section of the "Distributed Git - Maintaining a Project" chapter in ProGit 2
* [Documentation/technical](https://github.com/git/git/tree/master/Documentation/technical) folder which has technical information about various internal things.

**Comment by chriscool** [comment]
Yeah, I would be ok with adding those. The only thing left to do is to find good section titles to put them in :-)
